mythril/laser/ethereum/instructions.py

```diff
@@ -984,14 +1045,13 @@ def codecopy_(self, global_state: GlobalState) -> List[GlobalState]:
                 )
             return [global_state]
```

Comment:
Very minor note: The return value in `_code_copy_helper` is unused. We can do `return self._code_copy_helper(...)` in `codecopy` and `extcodecopy` or we can delete the return values in `_code_copy_helper` and save a line or two of code. Or we can just leave things as is.